x-pack/legacy/plugins/infra/public/pages/logs/analysis/sections/anomalies/index.tsx

```diff
@@ -0,0 +1,211 @@
+              })}
+            </h2>
+          }
+          titleSize="m"
+          body={
+            <p>
+              {i18n.translate('xpack.infra.logs.analysis.anomalySectionNoDataBody', {
+                defaultMessage: 'You may want to adjust your time range.',
+              })}
+            </p>
+          }
+        />
+      ) : !hasAnomalies ? (
+        <EuiEmptyPrompt
+          title={
+            <h2>
+              {i18n.translate('xpack.infra.logs.analysis.anomalySectionNoAnomaliesTitle', {
+                defaultMessage: 'No anomalies were detected.',
+              })}
+            </h2>
+          }
+          titleSize="m"
+        />
+      ) : (
+        <>
+          <EuiFlexGroup>
+            <EuiFlexItem grow={8}>
+              <AnomaliesChart
+                chartId="overall"
+                setTimeRange={setTimeRange}
+                timeRange={timeRange}
+                series={logEntryRateSeries}
+                annotations={anomalyAnnotations}
+                renderAnnotationTooltip={renderAnnotationTooltip}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem grow={2}>
+              <EuiStat
+                title={numeral(results.totalNumberOfLogs).format('0.00a')}
+                description={i18n.translate(
+                  'xpack.infra.logs.analysis.overallAnomaliesNumberOfLogsDescription',
+                  {
+                    defaultMessage: 'Number of logs',
+                  }
+                )}
+                reverse
+              />
+              <EuiStat
+                title={Number(topAnomalyScore).toFixed(0)}
+                description={i18n.translate(
+                  'xpack.infra.logs.analysis.overallAnomaliesTopAnomalyScoreDescription',
+                  {
+                    defaultMessage: 'Max anomaly score',
+                  }
+                )}
+                reverse
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="l" />
+          <AnomaliesTable results={results} setTimeRange={setTimeRange} timeRange={timeRange} />
+        </>
+      )}
+    </>
+  );
+};
+
+interface ParsedAnnotationDetails {
+  anomalyScoresByPartition: Array<{ partitionId: string; maximumAnomalyScore: number }>;
+}
+
+const overallAnomalyScoreLabel = i18n.translate(
+  'xpack.infra.logs.analysis.overallAnomalyChartMaxScoresLabel',
+  {
+    defaultMessage: 'Max anomaly scores:',
+  }
+);
+const AnnotationTooltip: React.FunctionComponent<{ details: string }> = ({ details }) => {
+  const parsedDetails: ParsedAnnotationDetails = JSON.parse(details);
+  return (
+    <div>
+      <span>{overallAnomalyScoreLabel}</span>
+      <ul>
+        {parsedDetails.anomalyScoresByPartition.map(
+          ({ partitionId, maximumAnomalyScore }, index) => {
+            return (
+              <li
+                key={`${index}-overall-anomaly-chart-${partitionId}-partition-score-${maximumAnomalyScore}`}
```

Comment:
Not that it would come up a lot, but the score should probably not be part of the key so the element can be updated when the score changes instead of being recreated. Similarly, the `index` might be counter-productive since the `partitionId` should be unique within this set of children.